terraform_egress_pub/cloudfront.tf

```diff
@@ -7,7 +7,23 @@ data "aws_acm_certificate" "rules_cert" {
   domain      = var.distribution_domain
   most_recent = true
   statuses    = ["ISSUED"]
-  types       = ["IMPORTED"]
+  types       = ["AMAZON_ISSUED"]
+}
+
+/* A Lambda@Edge for injecting security headers */
+module "security_header_lambda" {
+  source = "transcend-io/lambda-at-edge/aws"
+  # We are stuck with this ancient version until we upgrade to
+  # Terraform version 0.13 or higher.
+  version = "0.0.2"
+
+  description            = "Adds HSTS security header to the response"
+  lambda_code_source_dir = "${path.root}/add_security_headers"
+  name                   = "add_security_headers"
+  # TF 0.12 doesn't know about 14.x, so we're stuck with 12.x until we
+  # can upgrade.
+  runtime = "nodejs12.x"
```

Comment:
Is this actually a Terraform limitation? I'm not sure if it's a problem with the module, the Terraform version, or the AWS  provider version.

Reply:
You're right, it's actually a limitation of the AWS provider:
https://github.com/hashicorp/terraform-provider-aws/blob/2f23a59662cc53a1c9403517dc1312d2846b5434/aws/resource_aws_lambda_function.go#L26-L48

Reply:
Thanks for taking the time to look into where the limitation was to be found. I believe an AWS provider v3 upgrade is likely a lot less potential work than upgrading Terraform versions.

Reply:
I can't remember- is there any reason why we never upgraded to version 3.x of the AWS provider in this repo?  We took care of the rest of our repos with https://github.com/cisagov/cool-system/issues/93.

Reply:
I'm not sure.  I'm OK upgrading just the `terraform_egress_pub` directory, but I wasn't sure if @mcdonnnj wanted to keep all of this repo at the same AWS provider version.

Reply:
See commits 8868b16 and 48be7ab.

Reply:
Excellent!